<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>638</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>945</width>
    <height>638</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>945</width>
    <height>638</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(24, 24, 24);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/Logo_2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="usuario1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
font: 16pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Esperando</string>
    </property>
   </widget>
   <widget class="QLabel" name="usuario2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 16pt &quot;Kristen ITC&quot;;
background-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Esperando</string>
    </property>
   </widget>
   <widget class="QLabel" name="usuario3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 16pt &quot;Kristen ITC&quot;;
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Esperando</string>
    </property>
   </widget>
   <widget class="QLabel" name="usuario4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);
font: 16pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Esperando</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>ingresar_usuario()</slot>
 </slots>
</ui>
